services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceControllerMultiClusterConfig.java

```diff
@@ -283,6 +283,10 @@ public long getServiceDiscoveryRegistrationRetryMS() {
     return getCommonConfig().getServiceDiscoveryRegistrationRetryMS();
   }
 
+  public String getControllerResourceInstanceGroupTag() {
+    return getCommonConfig().getControllerResourceInstanceGroupTag();
+  }
```

Comment:
We should not use this from common configs. This needs to be set at a per-cluster level